<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserDialog</class>
 <widget class="QDialog" name="AddUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QPushButton" name="addUserButton">
     <property name="text">
      <string>Add user</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>User name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="passwordEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="generatButton">
     <property name="text">
      <string>Generate password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="userNameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="repeatLabel">
     <property name="text">
      <string>Repeat password:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="repeatEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="adminRadioButton">
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="memberRadioButton">
     <property name="text">
      <string>Memeber</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
